### ISSUE TITLE
fix(a11y): text color contrast

### ISSUE THREAD

**Issue by mfranzke** [calcom/cal.com#17165]
### Issue Summary

The signing up terms textual paragraph get's an 80% opacity defined on BookEventForm, that for this text colors contrast violates [Success Critera 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) that expects a color contrast of at least 4.5:1

### Steps to Reproduce

1. Go to a book event form.
2. Select a time. You should get to confirmation page on which you are requested to provide your name and email, compare to the following screenshot:
<img width="386" alt="image" src="https://github.com/user-attachments/assets/b8801c5a-d5e8-41de-9885-54934b255321">

### Actual Results

On the confirmation page where you enter your name, you'll recognize that the signing in terms text has a color contrast which is too low.

### Expected Results

The text should be accessible.

### Technical details

independent from browser environment.

### Evidence

Run Google Lighthouse out of Google Web Developer tools to check for any reportings on this topic.
